template.md

```diff
@@ -0,0 +1,305 @@
+---
+# required metadata
+
+title: [ARTICLE TITLE | SERVICE NAME]
+description:
+keywords:
+author: [GITHUB USERNAME]
+manager: [MANAGER ALIAS]
```

Comment:
I'll change this to wpickett for now. They're working on a way to automate this internally.